tests/app/test_notifying.py

```diff
@@ -208,6 +210,8 @@ def test_notifier():
         notes = notifier.getNotes()
         assert len(notes) == 3
 
+        assert notes[2].datetime[-6:] == "+00:00"
```

Comment:
Instead of just truncating, using `mockHelpingNowUTC` and asserting the full timestamp.

Reply:
I didn't know about `mockHelpingNowUTC`. Code updated with the mock.